ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiUserManagementService.scala

```diff
@@ -349,6 +390,33 @@ private[apiserver] final class ApiUserManagementService(
         Future.successful(t)
     }
 
+  private def verifyPartyExistInIdp(
+      rights: Set[UserRight],
+      identityProviderId: IdentityProviderId,
+      isParticipantAdmin: Boolean,
+  ): Future[Unit] =
+    if (isParticipantAdmin) Future.unit
```

Comment:
I thought we wanted to require that the parties exist in all cases (breaking change) also for the participant admins

Reply:
Let's discuss it offline tomorrow. I would still do it as a separate PR not to extend this one.

Reply:
As agreed - we do it as a separate step